app/controllers/robots_controller.rb

```diff
@@ -0,0 +1,13 @@
+class RobotsController < ApplicationController
+  layout :none
+
+  def show
+    render plain: <<~ROBOTS
```

Comment:
Worth a comment to say this is so we can redirect requests to the robots.txt on the beta domain to avoid them getting indexed?

Reply:
Good point, will do.